libs/core/src/lib/menu/menu.component.ts

```diff
@@ -129,6 +129,10 @@ export class MenuComponent implements MenuInterface, AfterContentInit, AfterView
     @Output()
     readonly activePath: EventEmitter<MenuItemComponent[]> = new EventEmitter<MenuItemComponent[]>();
 
+    /** Emits array of active menu items */
```

Comment:
Shouldn't it be `Emits event, when item is selected`?

Reply:
Not in basic menu component but in the split button component, which listens for the menu item which is selected. Because this feature is for split button not menu

Reply:
This doesn't emit an array as the comment suggests